qa-core/src/account-api/tests/accounts/accounts.cloud.spec.ts

```diff
@@ -42,9 +45,15 @@ describe("Accounts", () => {
       // Send the verification request
       await config.accountsApi.accounts.verifyAccount(code!)
 
+      // Verify self response is unauthorized
+      await config.api.accounts.self({ status: 403 })
+
       // Can now log in to the account
       await config.loginAsAccount(createAccountRequest)
 
+      // Verify self response matches account
+      await config.api.accounts.self()
```

Comment:
We'll want to do some assertion here, e.g. that the email of the self response matches the email of the created account